docs/pages/guides/view-transition.md

```diff
@@ -0,0 +1,622 @@
+              "type": "rescaler",
+              "id": "rescaler_1",
+              // highlight-start
+              "width": 480,
+              // highlight-end
+              "child": { "type": "input_stream", "input_id": "input_1" },
+            }
+          ]
+        }
+      }
+    }
+    ```
+
+    A few seconds latter update a scene with a different `width`:
+    ```http
+    POST: /api/output/output_1/update
+    Content-Type: application/json
+
+    {
+      "video": {
+        "root": {
+          "type": "view",
+          "background_color_rgba": "#4d4d4dff",
+          "children": [
+            {
+              "type": "rescaler",
+              "id": "rescaler_1",
+              // highlight-start
+              "width": 1280,
+              "transition": { "duration_ms": 2000 },
+              // highlight-end
+              "child": { "type": "input_stream", "input_id": "input_1" },
+            }
+          ]
+        }
+      }
+    }
+    ```
+  </TabItem>
+  <TabItem value="membrane" label="Membrane Framework">
+    Set initial scene for the transition and after few seconds update a component
+    with a different `width`:
+    ```elixir
+    def handle_setup(ctx, state) do
+      request = %LiveCompositor.Request.UpdateVideoOutput{
+        output_id: "output_1",
+        root: %{
+          type: :view,
+          background_color_rgba: "#4d4d4dff",
+          children: [
+            %{
+              type: :rescaler,
+              id: "rescaler_1",
+              // highlight-start
+              width: 480,
+              // highlight-end
+              child: %{ type: :input_stream, input_id: :input_1 },
+            }
+          ]
+        }
+      }
+      {[
+        notify_child: {:live_compositor, request},
+        start_timer: {:add_videos_timer, Membrane.Time.seconds(2)}
+      ], state}
+    end
+
+    def handle_tick(:start_transition, _ctx, state)
+      request = %LiveCompositor.Request.UpdateVideoOutput{
+        output_id: "output_1",
+        root: %{
+          type: :view,
+          background_color_rgba: "#4d4d4dff",
+          children: [
+            %{
+              type: :rescaler,
+              id: "rescaler_1",
+              // highlight-start
+              width: 1280,
+              transition: %{ duration_ms: 2000 },
+              // highlight-end
+              child: %{ type: :input_stream, input_id: :input_1 },
+            }
+          ]
+        }
+      }
+      {[
+        notify_child: {:live_compositor, request},
+        stop_timer: :start_transition
+      ], state}
+    end
+    ```
+  </TabItem>
+</Tabs>
+
+In the first update request you can see that rescaler has width of 480, and in the second one it is changed
+to 1280 and `transition.duration_ms: 2000` was added.
+
+It is important that component has the same `"id"` in both initial state and the update that starts the
+transition, otherwise it will switch immediately to the new state without a transition.
```

Comment:
```suggestion
In the first update request, you can see that the rescaler has a width of 480, and in the second one, it is changed
to 1280 and `transition.duration_ms: 2000` was added.

The component must have the same `"id"` in both the initial state and the update that starts the
transition, otherwise it will switch immediately to the new state without a transition.
```